mediaqueries/Overview.bs

```diff
@@ -1788,6 +1796,16 @@ Rare Interaction Capabilities: the 'any-pointer' and 'any-hover' features</h3>
 		or to hide them altogether to reduce visual clutter if ''(any-pointer: fine)'' is false.
 	</div>
 
+	<div class="example">
+	    On a touchscreen laptop, the built-in trackpad is usually treated as the primary
+	    input mechanism. As such, ''(hover: hover)'' is true. However, due to the presence of the
+	    touchscreen, ''(any-hover: none)'' is also true. An author may decide that in this case,
+	    the page should not rely on hover capabilities, as a user may choose to
+	    use the non-hover-capable input mechanism. On the other hand, an author may decide that
+	    use of the hover capability is essential for their page, and display a notification
+	    reminding users to only use their mouse/trackpad.
+	</div>
+
```

Comment:
Same comment as above:  `any-hover: none` only matches if none of the devices can hover, not merely if some of them cannot, so this example is incorrect.